source/whats-new.txt

```diff
@@ -28,8 +28,28 @@ Learn what's new in:
 What's New in 4.6
 -----------------
 
-See `v4.6.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v4.6.0>`__
-on GitHub.
+New features of the 4.6 Node.js driver release include:
+
+TypeScript: ``ChangeStreamDocument``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Ability to define your own custom type for the top level document
+  returned in a change event.
+- The ``ChangeStreamDocument`` type now defaults to union all the
+  possible events.
+- Added `the following properties
+  <{+api+}/modules.html#ChangeStreamDocument>`__ : ``to``, 
+  ``lsid``, ``txnNumber`` and ``clusterTime``.
```

Comment:
Issue: this doesn't seem like something we can expect the reader to understand what the fields are used for without some context.
Suggestion: it appears these are documented in the Server manual, so linking to them would be helpful.